packages/expo-font/src/ExpoFontLoader.web.js

```diff
@@ -1,27 +1,43 @@
 // @flow
 
-import WebFont from 'webfontloader';
+function isFunction(method) {
+  return method && typeof method === 'function';
+}
+
+function createWebStyle(fontFamily, resource) {
+  const fontStyle = `@font-face {
+    font-family: ${fontFamily};
+    src: url(${resource});
+  }`;
+
+  const styleElement = document.createElement('style');
+  styleElement.type = 'text/css';
+  if (styleElement.styleSheet) {
+    styleElement.styleSheet.cssText = fontStyle;
+  } else {
+    const textNode = document.createTextNode(fontStyle);
+    styleElement.appendChild(textNode);
+  }
+  return styleElement;
+}
 
 export default {
   get name(): string {
     return 'ExpoFontLoader';
   },
-  loadAsync(fontFamilyName: string, fontUri: string): Promise {
-    return new Promise((resolve, reject) => {
-      WebFont.load({
-        active: resolve,
-        inactive() {
-          reject(
-            new Error(
-              `ExpoFontLoader.loadAsync(): The browser does not support linked fonts or the font ${fontFamilyName} could not be loaded from: ${fontUri}`
-            )
-          );
-        },
-        custom: {
-          families: [fontFamilyName],
-          urls: [fontUri],
-        },
-      });
-    });
+  loadAsync(fontFamilyName: string, resource: string): Promise {
+    const canInjectStyle =
+      document &&
+      document.head &&
+      isFunction(document.head.appendChild) &&
```

Comment:
Typically `document.head.appendChild &&` is enough here for feature detection. That said, I think these checks aren't necessary since all modern browsers support this. Is there a specific case you are trying to defend against?